skbio/stats/distance/tests/test_base.py

```diff
@@ -467,6 +468,32 @@ def test_to_data_frame_default_ids(self):
                            index=['0', '1'], columns=['0', '1'])
         assert_data_frame_almost_equal(df, exp)
 
+    def test_to_series_1x1(self):
+        series = self.dm_1x1.to_series()
+        exp = pd.Series([0.0],
+                        pd.MultiIndex(levels=[['a'], ['a']],
+                        labels=[[0], [0]]))
+        exp = exp[exp > 0]
+        assert_series_almost_equal(series, exp)
+
+    def test_to_series_3x3(self):
+        series = self.dm_3x3.to_series()
+        exp = pd.Series([0.0, 0.01, 4.2, 0.01, 0.0, 12.0, 4.2, 12.0, 0.0],
```

Comment:
My new tests don't seem to be reading the index in the way you suggested.  The assert_series_almost_equal is expecting the MultiIndex levels and labels.  I may have implemented incorrectly.